README.md

```diff
@@ -23,6 +23,17 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
+Register at [ReCAPTCHA](http://www.google.com/recaptcha/admin) using the following configuration.
+|Field       |Value      |
+|--------------|-----------|
+|Label         |short      |
+|reCAPTCHA type|reCAPTCHAv2|
+|Domains       |localhost  |
+|--------------|-----------|
+
+Once registered copy the `RECAPTCHA_SECRET_KEY` to the env file
+and set the `RECAPTCHA_SITE_KEY` env variable within the build docker image step.
```

Comment:
Can this go inside the `.env` file as well and we source it before running the docker build command?

Reply:
I think we already have it in `.env`
<img width="466" alt="Screen Shot 2019-10-08 at 9 24 55 PM" src="https://user-images.githubusercontent.com/3537801/66451673-594c6300-ea12-11e9-8438-f7dfa61772d7.png">

Reply:
I was specifically talking about the `RECAPTCHA_SITE_KEY` since it's used in the docker build step after creating the .env file.
```GRAPHQL_BASE_URL=http://localhost:8080 \
HTTP_API_BASE_URL=http://localhost \
RECAPTCHA_SITE_KEY=your_recaptcha_site_key \
docker build -t local/short:latest .

Reply:
Hi  @shade34321, `RECAPTCHA_SITE_KEY` should be added here: https://github.com/byliuyang/short/blob/master/app/adapter/web/.env.development

 The `.env` file is used for `docker-compose` phase to launch the application.

Reply:
Ok, added it in.

Reply:
> Ok, added it in and called it `REACT_APP_RECAPTCHA_SITE_KEY` to conform to other variables in that file and what you called it in another comment.

Hi @shade34321, we have to name the environmental variable `REACT_APP_xxxx` for `create-react-app` to recognize it